pipelines/h6c/rtp/v2/h6c_rtp_stage_1.toml

```diff
@@ -123,6 +123,8 @@ actions = ["SETUP",
            "LIBRARIAN_SSINS",
            "RFI_INSPECT_NOTEBOOK",
            "DELAY_SPECTRUM_INSPECT_NOTEBOOK",
+           "ALL_SKY_IMAGE_SNAPSHOT",
+           "ALL_SKY_IMAGE_MOVIE",
```

Comment:
You're going to want to make sure CLEANUP doesn't run until ALL_SKY_IMAGE_MOVIE does.

Reply:
done